packages/@okta/vuepress-site/docs/guides/step-up-authentication/main/index.md

```diff
@@ -0,0 +1,171 @@
+
+In Okta Classic Engine when a user doesn't have a session, the more restrictive policy is evaluated first. If the Okta sign-on policy is more restrictive, then that is evaluated first. Otherwise, the application sign-on policy is evaluated. When an ACR value is passed in the authorize request and the application sign-on policy is more restrictive, the application sign-on policy is evaluated. If it isn’t more restrictive, then the `acr_values` parameter in the request is evaluated. When a user already has a session, the application sign-on policy is always applied first. Then, if the application sign-on policy requirements are satisfied, the `acr_values` parameter in the request is evaluated.
+
+In both Identity Engine and Classic Engine, if the user has a session, the previously satisfied authenticators are considered before prompting for factors that are required by the `acr_values` parameter in the request. Additionally, if the user is unable to satisfy the level of assurance, Okta returns an [error](https://openid.net/specs/openid-connect-unmet-authentication-requirements-1_0.html) (`error=unmet_authentication_requirements`) to the callback endpoint.
+
+### Factor enrollment
+
+If the user hasn't enrolled a factor, during step-up authentication the user is prompted to enroll a factor as long as an [authenticator enrollment policy](/docs/reference/api/policy/#authenticator-enrollment-policy) is configured (or the [Multifactor Enrollment Policy](/docs/reference/api/policy/#multifactor-mfa-enrollment-policy) in Classic Engine).
+
+## Predefined parameter values
+
+The following predefined optional parameters are available for use in your authorization requests:
+
+> **Note:** Any unsupported `acr_values` sent in the authorize request are ignored.
+
+| Parameter value<br> for `acr_values` | Description           | Parameter Type   | DataType   |
+| :----------------------- | :----------------------------------------- | :--------------- | :--------- |
+| `urn:okta:loa:1fa:any`   | Any one factor. Allows one factor authentication with no requirements on which factor. | Query | String |
+| `urn:okta:loa:2fa:any`   | Any two factors. Allows two factor authentication with no requirements on which factors. | Query | String |
+| `urn:okta:loa:1fa:pwd`   | Password only. Allows one factor authentication that requires the user’s password. | Query | String |
+| `urn:okta:loa:2fa:any:ifpossible` | Any two factors if possible. Allows two factor authentication with no requirements on which factors. Any two factors are presented only if the user is enrolled, otherwise any one factor is presented. | Query | String |
```

Comment:
```suggestion
| `urn:okta:loa:2fa:any:ifpossible`<br><br><ApiLifecycle access="ie" /> | Any two factors if possible. Allows two factor authentication with no requirements on which factors. Any two factors are presented only if the user is enrolled, otherwise any one factor is presented. | Query | String |
```